OptimizelySDK/Bucketing/DecisionService.cs

```diff
@@ -256,8 +256,8 @@ public void SaveVariation(Experiment experiment, Variation variation, UserProfil
         /// <param name = "userId" >User Identifier</param>
         /// <param name = "filteredAttributes" >The user's attributes. This should be filtered to just attributes in the Datafile.</param>
         /// <returns>null if the user is not bucketed into the rollout or if the feature flag was not attached to a rollout.
-        /// otherwise the Variation the user is bucketed into</returns>
-        public virtual Variation GetVariationForFeatureRollout(FeatureFlag featureFlag, string userId, UserAttributes filteredAttributes)
+        /// otherwise the FeatureDecision entity</returns>
+        public virtual FeatureDecision GetVariationForFeatureRollout(FeatureFlag featureFlag, string userId, UserAttributes filteredAttributes)
         {
             if (string.IsNullOrEmpty(featureFlag.RolloutId))
```

Comment:
Please add condition featureFlag == null

Reply:
#L272 please add the same for rollout.

Reply:
Added for feature flag. We do not need it for rollout as ProjectConfig.GetRolloutFromId() returns empty rollout object in that case.